app/invocation/invocation_targets.tsx

```diff
@@ -91,6 +96,21 @@ export default class TargetsComponent extends React.Component {
             model={this.props.model}
             filter={this.props.filter}
             pageSize={this.props.pageSize}
+            skipped={false}
+          />
+        )}
+
+        {!!this.props.model.skipped.length && (
+          <TargetsCardComponent
+            buildEvents={this.props.model.skipped}
+            className="card-neutral"
+            iconPath="/image/info.svg"
+            presentVerb={`${this.props.model.skipped.length == 1 ? "target" : "targets"}`}
+            pastVerb={`${this.props.model.skipped.length == 1 ? "target" : "targets"} skipped`}
+            model={this.props.model}
+            filter={this.props.filter}
+            pageSize={this.props.pageSize}
+            skipped={true}
```

Comment:
Instead of adding this `skipped` property on every card, what do you think about doing what all of the other cards do and add a className called `card-skipped`. And use that to style it (instead of the "purple" class).

We can use yellow-700 color from https://materialui.co/colors which has the hex: #FBC02D

Reply:
IMO we should use purple to be consistent with how Bazel colors the "SKIPPED" label in the log output: 
![image](https://user-images.githubusercontent.com/2414826/126187973-82974cf0-2660-46b7-a9bf-153c0ba25195.png)

(Purple 500 is #9C27B0)

Reply:
That sounds good, we should still probably do it with a `card-skipped` class rather than special-casing skipped everywhere.